### PR TITLE
WIP: Custom perf test cases for private preview

### PR DESCRIPTION
**Pull request by vadim-kovalyov**
A set of tests to run before private preview for a longer duration.

### CODE REVIEW

File: builds/misc/templates/mqtt-perf-run.yaml [hunk 1]
```diff
@@ -4,262 +4,126 @@ jobs:
...
-      publishers: 2
-      subscribers: 10
+      parentJob: run_completed_10
+      id: 11
+      publishers: 10
+      subscribers: 1
       topics: 1
-      payload: 131072 # 128kb
-      qos: 1
+      payload: 1048576
```
Comment (varunpuranik) [inline builds/misc/templates/mqtt-perf-run.yaml:118]: 1Mb messags with 10 publishers and 1 sub? Will this run fine? Won't it eventually choke?

File: builds/misc/templates/mqtt-perf-run.yaml [hunk 2]
```diff
@@ -4,262 +4,126 @@ jobs:
...
-  
-  - template: mqtt-perf-test-case.yaml
-    parameters: 
-      parentJob: run_completed_252
-      id: 253
-      publishers: 2
-      subscribers: 10
-      topics: 10
-      payload: 131072 # 128kb
+      parentJob: run_completed_11
```
Comment (varunpuranik) [inline builds/misc/templates/mqtt-perf-run.yaml:123]: What is the diff between this and the one above?

Comment (varunpuranik) [reply builds/misc/templates/mqtt-perf-run.yaml:123]: Oh qos0 vs qos1.. got it

File: builds/misc/templates/mqtt-perf-run.yaml (hunk 1 above)
Comment (vadim-kovalyov) [reply builds/misc/templates/mqtt-perf-run.yaml:118]: We'll see :)

**Comment by vadim-kovalyov** [comment]
No longer needed.

**Pull request closed without merging**